cli/tests/activate.bats

```diff
@@ -2779,6 +2779,42 @@ EOF
   assert_line --partial "$EXPECTED_GIT"      # from inner
 }
 
+@test "profile: CMAKE_PREFIX_PATH not modified when cmake is not installed" {
+  project_setup
+
+  # Shouldn't be set by default.
+  run --separate-stderr "$FLOX_BIN" activate -- \
+    bash -c 'echo "CMAKE_PREFIX_PATH is: $CMAKE_PREFIX_PATH"'
+  assert_success
+  assert_output "CMAKE_PREFIX_PATH is: "
+
+  # Should respect existing variable from outside activation.
+  CMAKE_PREFIX_PATH="/fakepath" run --separate-stderr "$FLOX_BIN" activate -- \
+    bash -c 'echo "CMAKE_PREFIX_PATH is: $CMAKE_PREFIX_PATH"'
+  assert_success
+  assert_output "CMAKE_PREFIX_PATH is: /fakepath"
+}
+
+@test "profile: CMAKE_PREFIX_PATH is modified when cmake is installed" {
+  # We don't need an environment, but we do need wait_for_watchdogs to have a
+  # PROJECT_DIR to look for
+  project_setup_common
+
+  "$FLOX_BIN" init
+  _FLOX_USE_CATALOG_MOCK="$GENERATED_DATA/resolve/cmake-gnumake.yaml" \
+    run "$FLOX_BIN" install cmake
```

Comment:
The tests currently fail because the install command doesn't match the mock.
```suggestion
  _FLOX_USE_CATALOG_MOCK="$GENERATED_DATA/resolve/cmake-gnumake.yaml" \
    run "$FLOX_BIN" install cmake gnumake
```

I seem to recall that a previous iteration of the mock didn't have `gnumake`. Why do we need both?

Reply:
... and forgot to mention that I updated the command to match. You're right that previous version didn't install gnumake, and tests passed, so I'm confident tests will pass now.